WorkspaceServer.Tests/Kernel/KernelClientTests.cs

```diff
@@ -122,14 +160,7 @@ public async Task Kernel_can_pound_r_nuget_using_kernel_client()
             var reader = new StreamReader(output, Encoding.UTF8);
 
             var text = reader.ReadToEnd();
-            var events = text.Split(new[] { Environment.NewLine }, StringSplitOptions.RemoveEmptyEntries)
-                .Select(JsonConvert.DeserializeObject<StreamKernelEvent>).ToList();
-
-            events.Should()
-                .Contain(e => e.EventType == nameof(NuGetPackageAdded));
-
-            events.Should()
-                .NotContain(e => e.EventType == nameof(CommandNotRecognized));
+            this.Assent(text, _configuration);
```

Comment:
This might be overly strict. Assent should be used for tests that are explicitly contract tests. This was (until now, and in terms of naming) a test of functionality. We should separate these so test failures are more informative and functional tests are resilient to normal refactoring.